jitbuilder/apigen/cppgen.py

```diff
@@ -889,6 +896,9 @@ def write_class_header(self, writer, class_desc, namespaces, class_names):
         writer.write("#ifndef {}_INCL\n".format(class_desc.name()))
         writer.write("#define {}_INCL\n\n".format(class_desc.name()))
 
+        for header in self.get_common_system_includes():
+            writer.write(self.generate_include(header))
```

Comment:
Might be good to add a `writer.write("\n")` after the loop to aid in the readability of the generated code.

Reply:
I'm actually relying on line 907, just below, to output a newline after all the header includes.